src/pages/careers/index.tsx

```diff
@@ -31,7 +32,7 @@ const Careers = () => (
         <LifeAtDerivCarousel />
         <EmployeeTestimonial />
         <OurHiringProcess />
-        {/* <BeSquareBanner /> */}
+        <BeSquareBanner />
```

Comment:
Is this change part of this PR?

Reply:
It's not my own PR but seems to me that yes